src/uikit/Typography.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'react-emotion';
+
+
+const HeadingBaseStyles = `
+  
+`;
+
+export const HeadingH2 = styled('h2')`
```

Comment:
I really like this idea of having componentized theme elements, something like the following:
```
export const P = styled('p')`...`;

export const H1 = styled('h1')`...`;

...
```

if we do that for all elements, and instead of importing the theme into CSS in the JSX we can just import these components and use them throughout the app